dada2_filter.R

```diff
@@ -0,0 +1,303 @@
+#!/usr/bin/env Rscript
+
+# Read in package to read in command-line options.
+library("optparse")
+
+# Source Rscript containing functions 
+
+version <- "1.0"
+
+option_list <- list(
+
+  make_option(c("-f", "--f_path"), type="character", default=NULL,
+              help="Location of forward reads (required)." , metavar="path"),
+  
+  make_option(c("-r", "--r_path"), type="character", default=NULL,
+              help="Location of reverse reads (if applicable). Same as forward reads by default.",
+              metavar="path"),
+  
+  make_option(c("-s", "--single"), action = "store_true", type="logical", default=FALSE,
```

Comment:
I'd prefer keeping it as an option just in case there's an issue where users' reverse reads don't match the specified reads so they're erroneously treated as single-end.